test/io-tests.sh

```diff
@@ -186,16 +181,16 @@ ensureIatClaimWorks(){
   pgrStop
 }
 
-# ensure app settings don't reset on pool timeout of 10 seconds, see https://github.com/PostgREST/postgrest/issues/1141
+# ensure app settings don't reset on pool timeout, see https://github.com/PostgREST/postgrest/issues/1141
+# pool timeout set to 1s to shorten runtime
 ensureAppSettings(){
   pgrStart "./configs/app-settings.config"
   while pgrStarted && test "$( rootStatus )" -ne 200
   do
     # wait for the server to start
-    sleep 0.1 \
-    || sleep 1 # fallback: subsecond sleep is not standard and may fail
+    sleep 0.1
   done
-  sleep 11
+  sleep 2
```

Comment:
@steve-chavez Can you have a good look at this? You wrote the test.

I changed the pool timeout to 1s and the sleep to 2s. Was 10s / 11s before. This should achieve the same, but doesn't waste 9s of my life each time the io tests are re-run :D. Does that make sense?

Reply:
Sure! I just did a `postgrest-test-io` and it worked good.

Changing the pool timeout is much better. Don't know why I didn't thought of that before, I guess because it only wasted my own time  :laughing:

Reply:
You know - I don't think I would have even noticed without that nice `postgrest-watch postgrest-test-io` infinite loop I fixed here as well. But that kind of stuck out, because it was taking 10 seconds for this test - and then immediately restarted :D.